scripts/eachdist.py

```diff
@@ -17,6 +17,8 @@
 DEFAULT_ALLSEP = " "
 DEFAULT_ALLFMT = "{rel}"
 
+NON_SRC_DIRS = ["build", "dist", "__pycache__", "lib", "venv", ".tox"]
```

Comment:
when trying to find python source files, we should not look in these directories.